src/middlewared/middlewared/plugins/audit/utils.py

```diff
@@ -6,7 +6,12 @@
 AUDIT_DATASET_PATH = '/audit'
 AUDITED_SERVICES = [('SMB', 0.1)]
 AUDIT_TABLE_PREFIX = 'audit_'
-AUDIT_LIFETIME = 7  # Temporary hard-coded retention until public APIs developed
+AUDIT_LIFETIME = 7
+AUDIT_DEFAULT_RESERVATION = 0
+AUDIT_DEFAULT_QUOTA = 0
+AUDIT_FILL_CRITICAL = 95
+AUDIT_FILL_WARNING = 80
+AUDIT_MIN_FREE = 1  # base minimum free size to keep availalable at all times
```

Comment:
Is `AUDIT_MIN_FREE` used anywhere?

If so, maybe fix the spelling "availalable".  If not, maybe remove.